src/content/guides/code-splitting.md

```diff
@@ -147,7 +145,7 @@ __webpack.config.js__
   };
 ```
 
-With the [`SplitChunks`](/plugins/split-chunks-plugin/) in place, we should now see the duplicate dependency removed from our `index.bundle.js` and `another.bundle.js`. The plugin should notice that we've separated `lodash` out to a separate chunk and remove the dead weight from our main bundle. Let's do an `npm run build` to see if it worked:
+With the [`optimization.SplitChunks`](/configuration/optimization/#optimization-splitchunks) configuration option in place, we should now see the duplicate dependency removed from our `index.bundle.js` and `another.bundle.js`. The plugin should notice that we've separated `lodash` out to a separate chunk and remove the dead weight from our main bundle. Let's do an `npm run build` to see if it worked:
```

Comment:
Why add `optimization`?

Reply:
by the text, `With X in place, we should see something removed...`, we dont put SplitChunksPlugin in place, we enable it via optimization configuration options...

Reply:
So i thought it is clearer to actually show users the configuration option. But your point is very correct in regard of wrong URL, it should be: `/plugins/split-chunks-plugin/#optimization-splitchunks` and not the `optimization` page

Reply:
there is also a conflict now, need to see what changed and resolve